main.tf

```diff
@@ -26,7 +26,7 @@ data "aws_iam_policy_document" "policy" {
       "ecr:PutImage",
     ]
     resources = [
-      "arn:aws:ecr:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:repository/${var.team_name}/${var.repo_name}",
+      "arn:aws:ecr:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:${var.team_name}/*",
```

Comment:
I think `repository/` is a required part of the ARN? https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arn-syntax-ecr

Reply:
this was rolled back as it was before, including repository